docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+### YAML Format
+
+Rasa Open Source 2.0 uses [YAML](https://yaml.org/spec/1.2/spec.html) as
```

Comment:
```suggestion
Rasa Open Source uses [YAML](https://yaml.org/spec/1.2/spec.html) as
```

I don't think we need to mention that if we add a note about the deprecated formats